tests/product/product.helper.js

```diff
@@ -194,11 +194,16 @@ const getProductById = async (id, operations) =>
       id,
     },
   });
-const deleteProduct = async (ids, operations) =>
+const deleteProducts = async (ids, operations) =>
   await operations.mutate({
     mutation: gql`
       mutation($ids: [ID!]) {
-        deleteProduct(ids: $ids) {
+        deleteProducts(ids: $ids) {
+          ... on Products {
+            items {
+              _id
+            }
```

Comment:
It seems to be ok, but why do we need to return an id of a deleted product? Do you handle it on the admin part somehow? Can you explain it?

Reply:
I thought it's not possible to use graphql mutation and not return any value. After some research I found that it can be done, but I'm not sure that it worth it. I mean using a void return type.(at least i don't know other ways how to do it)